src/components/Questionnaire/QuestionRenderer.tsx

```
@@ -0,0 +1,72 @@
+    } else {
+      newResponses.push(updatedResponse);
+    }
+    onResponseChange(newResponses);
+  };
+
+  return (
+    <div className="space-y-4">
+      {questions.map((question) => (
+        <div
+          key={question.id}
+          ref={(el) => (questionRefs.current[question.id] = el)}
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)